password-hash/src/b64.rs

```diff
@@ -0,0 +1,321 @@
+//! "B64" encoding.
+//!
+//! Subset of the standard Base64 encoding (RFC 4648, section 4) which omits
+//! padding (`=`) as well as extra whitespace, as described in the PHC string
+//! format specification:
+//! <https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md#b64>
+//!
+//! Supports the Base64 character subset: `[A-Z]`, `[a-z]`, `[0-9]`, `+`, `/`
```

Comment:
@newpavlov I just noticed you own the `b64` crate.

It might make sense to split this functionality out into its own crate (which could be used, for example, to implement PEM encoding in the `pkcs8` crate)